public/locales/es/login.json

```diff
@@ -1,6 +1,8 @@
 {
     "button": {
-        "login": "Iniciar sesión"
+        "login": "Iniciar sesión",
+        "error": "Error, prueba con otro código",
```

Comment:
this should just be error since it is used across the repo for all buttons

Reply:
actually we have a generic.error message in gamepage and "error": "Error" in translations. So I will leave this since it matches the english translation